src/renderer/qssWidget/js/qssOfficeWidget.js

```diff
@@ -0,0 +1,391 @@
+        components: {
+            repeater: {
+                type: "gpii.psp.repeater",
+                container: "{office}.dom.menuControls",
+                options: {
+                    model: {
+                        disabled: "{office}.model.disabled",
+                        value: "{office}.model.setting.value",
+                        styles: "{office}.model.setting.styles"
+                    },
+                    modelRelay: {
+                        items: {
+                            target: "items",
+                            singleTransform: {
+                                type: "fluid.transforms.free",
+                                func: "gpii.qssWidget.office.getRepeaterItems",
+                                args: [
+                                    "{office}.model.setting"
+                                ]
+                            }
+                        }
+                    },
+                    dynamicContainerMarkup: {
+                        container: "<div role=\"radio\" class=\"%containerClass fl-qssWidgetMenu-item fl-focusable\" tabindex=\"0\"></div>",
+                        containerClassPrefix: "flc-qssWidgetMenu-item"
+                    },
+                    handlerType: "gpii.qssWidget.office.presenter",
+                    markup: null,
+                    styles: {
+                        disabled: "disabled"
+                    },
+                    events: {
+                        onItemFocus: null
+                    },
+                    listeners: {
+                        "onCreate.enable": {
+                            this: "{that}.container",
+                            method: "removeClass",
+                            args: ["{that}.options.styles.disabled"]
+                        }
+                    }
+                }
+            },
+            closeTimer: {
+                type: "gpii.app.timer",
+                options: {
+                    listeners: {
+                        "onTimerFinished.closeWidget": {
+                            func: "{qssWidget}.close",
+                            args: ["{office}.keyboardEvent"]
+                        }
+                    }
+                }
+            }
+        },
+        listeners: {
+            "onCreate.visibilityChange": {
+                funcName: "gpii.qssWidget.office.addVisibilityChangeListener",
+                args: ["{closeTimer}"]
+            },
+            "onDestroy.visibilitychange": {
+                funcName: "gpii.qssWidget.office.removeVisibilityChangeListener"
+            }
+        },
+        invokers: {
+            calculateHeight: {
+                funcName: "gpii.qssWidget.calculateHeight",
+                args: [
+                    "{qssWidget}.container",
+                    "{that}.dom.menuControlsWrapper",
+                    "{that}.dom.heightListenerContainer"
+                ]
+            },
+            close: {
```

Comment:
This is now the 4th widget we have with a close timer and associated logic - please factor this up into a shared grade

Reply:
I agree with you, but maybe we can add a different JIRA task for that optimization?

Because this version of the Word Simplification is not gonna last, we are replacing the current version (a modified copy of the menuWidget), with fully custom two toggle widget either way. Not for this build, but we will replace it for sure.